.github/workflows/ci-terraform.yaml

```diff
@@ -23,3 +27,21 @@ jobs:
         run: |
           terraform init
           terraform validate
+
+      - name: 'Terraform - validate example-google-workspace'
+        working-directory: infra/example-google-workspace
+        run: |
+          terraform init
```

Comment:
Let's see how it grows, but maybe we could have a `terraformValidate.sh` shell/bash script and just do:

```
- name: 'Terraform - validate example-gcp-simple'
       working-directory: infra/example-gcp-simple
       run: ./scripts/terraformValidate.sh
```

To avoid repeting same pair of commands again and again.